### PR TITLE
Add options to prevent spamming Tuning Error messsages when not actually tuning or desired

### PR DESCRIPTION
**Pull request by Hwurzburg**
Prevents spamming tuning error messages if not actually actively tuning with TX tuning
Allows disabling messages entirely by setting error threshold to -1
Added metadata explanation of what message really means

and a drive-by  bug fix to the rms calculations for Pitch pids that has been there since 2016

### CODE REVIEW

**Review by IamPete1 — APPROVED** [review]

File: libraries/AP_Tuning/AP_Tuning.cpp [hunk 1]
```diff
@@ -49,7 +49,7 @@ const AP_Param::GroupInfo AP_Tuning::var_info[] = {
 
     // @Param: ERR_THRESH
     // @DisplayName: Controller error threshold
-    // @Description: This sets the controller error threshold above which an alarm will sound and a message will be sent to the GCS to warn of controller instability
+    // @Description: This sets the controller error threshold above which an alarm will sound and a message will be sent to the GCS to warn of controller instability while tuning. The error is the rms value of the P+D corrections in the loop. High values in hover indicate possible instability due to too high PID gains or excessively high D to P gain ratios.-1 will disable this message.
```
Comment (peterbarker) [inline libraries/AP_Tuning/AP_Tuning.cpp:52]: Do we really sound an alarm?!

Comment (Hwurzburg) [reply libraries/AP_Tuning/AP_Tuning.cpp:52]: do not know...dont have buzzers on my planes....

**Review by tridge — APPROVED** [review]

**Review by tridge — APPROVED** [review]

**Comment by tridge** [comment]
lgtm, @rmackay9 can you merge?

**Comment by rmackay9** [comment]
AP_Tuning is not used by Copter so no problem for me, merged!

**Pull request merged**